### ISSUE TITLE
liquid TypeError: key: expected bytes or bytearray, but got 'str' with JWT in Python3

### ISSUE THREAD

**Issue by vxnuke** [ccxt/ccxt#5340]
Hi just wondering if this project works with Python 3.7.3?
Code works fine in 2.7 but not 3.7.3 also tested with 3.6.8 seems to be python3 issue
Just having some issues. Here is a screenshot. Code is pretty basic
https://imgur.com/a/DPHIdS3

Code:
```
# -*- coding: utf-8 -*-

import os
import sys
import ccxt

# instantiate exchanges


liquid = ccxt.liquid({
    'apiKey': '',
    'secret': '',
})

try:
    # fetch account balance from the exchange
    liquidBalance = liquid.fetch_balance()

    # output the result
    print(liquidBalance)

except ccxt.DDoSProtection as e:
    print(type(e).__name__, e.args, 'DDoS Protection (ignoring)')
except ccxt.RequestTimeout as e:
    print(type(e).__name__, e.args, 'Request Timeout (ignoring)')
except ccxt.ExchangeNotAvailable as e:
    print(type(e).__name__, e.args, 'Exchange Not Available due to downtime or maintenance (ignoring)')
except ccxt.AuthenticationError as e:
    print(type(e).__name__, e.args, 'Authentication Error (missing API keys, ignoring)')
```
###################################################################
issue
###################################################################
```Traceback (most recent call last):
  File "balance.py", line 17, in <module>
    liquidBalance = liquid.fetch_balance()
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/liquid.py", line 270, in fetch_balance
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/base/exchange.py", line 421, in inner
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/base/exchange.py", line 450, in request
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/base/exchange.py", line 445, in fetch2
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/liquid.py", line 729, in sign
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/base/exchange.py", line 984, in jwt
  File "/usr/local/lib/python3.7/dist-packages/ccxt-1.18.787-py3.7.egg/ccxt/base/exchange.py", line 944, in hmac
  File "/usr/lib/python3.7/hmac.py", line 153, in new
    return HMAC(key, msg, digestmod)
  File "/usr/lib/python3.7/hmac.py", line 49, in __init__
    raise TypeError("key: expected bytes or bytearray, but got %r" % type(key).__name__)                                                                                                     
TypeError: key: expected bytes or bytearray, but got 'str'```

**Comment by kroitor** [comment]
> Hi just wondering if this project works with Python 3.7.3?

Yes, it does, however, there was a recent change regarding the dependencies for JWT, so the exchanges using JWT may be broken in Py3. Will take a look and will fix this shortly.

**Comment by kroitor** [comment]
Liquid is now fixed in Python3 as of version 1.18.788. Let us know if you still have difficulties with it after upgrading CCXT to the most recent version. Thx for reporting!

**Comment by vxnuke** [comment]
Liquid works great cheers :)
